### ISSUE TITLE
Custom .json files in complex_modifications are not listed

### ISSUE THREAD

**Issue by elkind00** [pqrs-org/Karabiner-Elements#2710]
I just updated to Karabiner-Elements 13.4.0 and discovered that my custom .json files in ~/.config/karabiner/assets/complex_modifications are no longer listed when I click on Add Rule.

**Comment by elkind00** [comment]
No response to this issue?

**Comment by MuhammedZakir** [comment]
Only _your_ `.json` files? Or all `.json` files in that directory? Does `Karabiner-Elements Preferences > Log` show any error?

**Comment by elkind00** [comment]
Thanks for the reply, much appreciated.  I could tell that this must work fine for you, so I played around with the content of my .json file, and after some changes it was once again listed when I clicked on Add Rule.

I think I see what must have happened.  Back in May when I updated to 13.4.0 I must have also made a minor change to my .json file that introduced a syntax error (now fixed apparently, but I don't know what it was).  I just now did a little testing by purposefully inserting syntax errors into my .json file, and it seems that the Add Rule button doesn't list .json files that have syntax errors, and that the Log tab doesn't include any information about these syntax errors.  Does that sound right?
